target/linux/ramips/image/rt288x.mk

```diff
@@ -12,11 +12,10 @@ define Build/gemtek-header
 endef
 
 define Device/airlink101_ar670w
-  BLOCKSIZE := 64k
   DEVICE_VENDOR := Airlink
   DEVICE_MODEL := AR670W
   IMAGE_SIZE := 3840k
-  KERNEL := $(KERNEL_DTB) | pad-to $$(BLOCKSIZE)
+  KERNEL := $(KERNEL_DTB) | pad-to $$$$(BLOCKSIZE)
```

Comment:
@hauke Sorry I made a mistake here. I didn't notice this error because my local build and ci automatically excluded this device. However, it will surely break the buildbot build.
ref: https://buildbot.staging.openwrt.org/images/#/builders/12/builds/4
I have sent a fix patch to the mailing list and cc to you.

BTW, I suggest marking the rt288x sub-target as source only. Because the only supported device Belkin F5D8235 has only 32MiB RAM, I don't think it can run the 5.15 kernel.